webrender/src/resource_cache.rs

```diff
@@ -317,6 +320,11 @@ impl ResourceCache {
             data: ImageData::new(bytes),
```

Comment:
The old image data is replaced by the new one, yes. The underlying buffer is reference counted, not sure what you mean by lost.